### DISCUSSION TITLE
Trouble to open documents in Word or Powerpoint

### DISCUSSION THREAD

**Discussion by Karo75** [chamilo/chamilo-lms#5599] [General]
Hello,

When I put a document in Word or Powerpoint in Chamilo (documents of learning path), I have to download the document before it opens. In pdf there is no problem. What can I do to open documents directly without downloading each time?

**Comment by Kaneda-1** [comment]
Hello @Karo75 

Chamilo allows the import of all types of documents (HTML, MS-Word®, MS-PowerPoint® MS-Excel®, PDF, OpenOffice, etc.). However, to open the documents directly in the browser, the user must have the necessary software installed on their computer.

For Word and PowerPoint documents, one option could be to convert them to PDF before uploading them to Chamilo, as you mentioned that PDF documents open directly without any problems.

Another option could be the installation of a plugin like OnlyOffice, which allows compatible documents to be opened directly within the learning path.

https://www.onlyoffice.com/en/office-for-chamilo.aspx
https://www.onlyoffice.com/blog/2023/11/onlyoffice-connector-v1-3-0-for-chamilo

**Comment by Moigaston** [comment]
Hello @Karo75, hello Kaneda-1,
As far as I know, it's not possible to open Word and PowerPoint documents with OnlyOffice directly within a learning path. OnlyOffice only works in the documents directory where it's installed.
In a learning path, you can only directly open PDFs. The only solution to open Word or PowerPoint documents in a learning path is to create a document in your learning path with a link to your Word or PowerPoint document, which are in your documents directory. However, you must install OnlyOffice and use the OnlyOffice link that allows you to open your Word or PowerPoint document in the documents directory.
